adafruit_wm8960/__init__.py

```diff
@@ -0,0 +1,415 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 Scott Shawcroft for Adafruit Industries
+# SPDX-FileCopyrightText: Copyright (c) 2024 Cooper Dalrymple
+#
+# SPDX-License-Identifier: MIT
+"""
+`adafruit_wm8960`
+================================================================================
+
+CircuitPython driver for WM8960 Stereo CODEC
+
+* Author(s): Scott Shawcroft, Cooper Dalrymple
+
+Implementation Notes
+--------------------
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://circuitpython.org/downloads
+
+# * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
+"""
+
+__version__ = "0.0.0+auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_WM8960.git"
+
+from busio import I2C
+from adafruit_simplemath import constrain, map_range
+
+from .advanced import *
+
+
+# pylint: disable=too-few-public-methods
+
+
+class Input:
+    """An enum-like class representing the input options of the WM8960. Used for
+    :attr:`WM8960.input`.
+    """
+
+    DISABLED = 0b000
+    """Disconnect all inputs."""
+
+    MIC1 = 0b001
+    """Connect input 1 to the microphone amplifier in single-ended mode."""
+
+    MIC2 = 0b011
+    """Connect input 1 and 2 to the microphone amplifier in differential mode."""
+
+    MIC3 = 0b101
+    """Connect input 1 and 3 to the microphone amplifier in differential mode."""
+
+    LINE2 = 0b010
+    """Connect input 2 as a line input."""
+
+    LINE3 = 0b100
+    """Connect input 3 as a line input."""
+
+
+# pylint: enable=too-few-public-methods
+
+
+class WM8960:
+    """Driver for controlling the WM8960 audio codec over an I2C connection.Used for receiving line
```

Comment:
```suggestion
    """Driver for controlling the WM8960 audio codec over an I2C connection. Used for receiving line
```

Reply:
Yeah, I noticed pre-commit is having issues. I'll run through and fix them this evening. I'm using a Debian machine and definitely struggling to get pre-commit running on my end (I believe this is a known issue related to disttools), otherwise I would have tried to catch that.